SnapshotTests/ThemeRemoteConfigurationTests.swift

```diff
@@ -0,0 +1,34 @@
+@testable import GliaWidgets
+import XCTest
+import SnapshotTesting
+
+final class ThemeRemoteConfigurationTests: SnapshotTestCase {
+    func test_themeForMessagesFromHistory() throws {
```

Comment:
So the idea is to make snapshots with the default theme and the same snapshots but with remote configuration?

Reply:
Yes, If needed we can capture default theme styling, then from file configuration. That will allow us keep changes in both themes.